app/src/features/authentication/login/LoginForm/LoginForm.tsx

```diff
@@ -103,18 +103,6 @@ export default function LoginForm() {
       onSettled: (_, error) => {
         setError(error ? error.message : "");
       },
-      onError: (error) => {
```

Comment:
Are you sure from your edits in this file? i think it could be out of the scope of your functionality. Just check you really mean these changes

Reply:
sorry, it was an error in merging